app/controllers/bars_controller.rb

```diff
@@ -1,3 +1,6 @@
+require "mailgun"
+require "dotenv"
+Dotenv.load
```

Comment:
Similarly, this does not have to be manually run. check the docs: https://github.com/bkeepers/dotenv